### PR TITLE
Refactor variables and text in oracle rpcs and dftxs

### LINKED ISSUE #514: Refactor variables and text in oracle rpcs and dftxs
#### What would you like to be added:
Should rename the following texts
1) APPLE to AAPL
2) TESLA to TSLA

Should rename the following variable
1) oracleid to oracleId

#### Why is this needed:
APPLE and TESLA are not acronyms
We use oracleId in whale project, not oracleid

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind refactor

#### What this PR does / why we need it:

Should rename the following texts

- APPLE to AAPL
- TESLA to TSLA

Should rename the following variable

- oracleid to oracleId

APPLE and TESLA are not acronyms
We use oracleId in whale project, not oracleid

#### Which issue(s) does this PR fixes?:

Fixes #514

### CODE REVIEW

**Review by canonbrother — APPROVED** [review]

**Comment by jingyi2811** [comment]
Fixing failed tests

**Comment by jingyi2811** [comment]
For those `oracleId` variable returned from C++ side, remain `oracleid` (All lowercase)

**Review by canonbrother — DISMISSED** [review]

File: website/docs/jellyfish/api/oracle.md [hunk 1]
```diff
@@ -110,11 +110,11 @@ Returns oracle data.
 
 ```ts title="client.oracle.getOracleData()"
 interface oracle {
-  getOracleData (oracleid: string): Promise<OracleData>
+  getOracleData (oracleId: string): Promise<OracleData>
 }
 
 interface OracleData {
-  oracleid: string
+  oracleId: string
```
Comment (fuxingloh) [inline website/docs/jellyfish/api/oracle.md:117]: eh no you can't edit interface coming out of `ain` you can only edit what you control

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: website/docs/jellyfish/api/oracle.md (hunk 1 above)
Comment (jingyi2811) [reply website/docs/jellyfish/api/oracle.md]: I knew. I have changed in the js file. Forgot to add in this md file.

**Review by fuxingloh — APPROVED** [review]

**Review by canonbrother — APPROVED** [review]

**Pull request merged**